src/core/scene/a-scene.js

```diff
@@ -320,7 +364,16 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+        // Postproc must be explicitly enabled by a component or system.
```

Comment:
The scene could check whether that post-processing component is attached to the scene rather than having to set it in the PP components. 

Though you'd need to add a standard post-processing component, but it might be easier to do on a separate PR and purely just add the tock logic in for now?